packages/odyssey-react-mui/src/themes/odyssey/palette.types.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare module "@mui/material/styles" {
+  interface PaletteColor {
+    lighter?: string;
```

Comment:
This may disappear before 1.0. I'm working with @josesolano-okta and Taylor to sort out what our schema should be.